com.unity.render-pipelines.universal/Runtime/ForwardLights.cs

```diff
@@ -43,9 +47,28 @@ static class LightConstantBuffer
 
         bool m_UseStructuredBuffer;
 
-        public ForwardLights()
+        bool m_UseClusteredRendering;
```

Comment:
Maybe we should isolate forward clustter lighting into other class? I fear that in future it could be a bit hard to distinguish which code is responsible for traditional forward and which one is for forward+. I had this issue when I was investigating HDRP decals where they had cluster and dbuffer solution combined into same class, it took me quite a time to find dbuffer code only

Reply:
Hmm yeah, we could do that. I think we should do that in another PR though. We would still need to use `ForwardLights` together with the other class though, as it shares all the GPU light data code.